dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -950,6 +952,166 @@ def _get_percentile(self, percentiles):
             quantiles[499] = median_value
         return quantiles
 
+    @property
+    def median_abs_deviation(self):
+        """
+        Get median absolute deviation.
+
+        :return: median absolute deviation
+        """
+        if not self._has_histogram:
+            return np.nan
+
+        bin_counts = self._stored_histogram['histogram']['bin_counts']
+        bin_edges = self._stored_histogram['histogram']['bin_edges']
+
+        # normalize bin counts
+        bin_counts = bin_counts.astype(float)
+        normalized_bin_counts = bin_counts / np.sum(bin_counts)
+
+        # calculate deviations from median
+        median = self.quantiles[499]
```

Comment:
this is not a guarantee, should use _get_percentile

Reply:
fixed with the new update